Jenkinsfile

```diff
@@ -35,7 +35,10 @@ pipeline {
 
     stage('Build and test package') {
       steps {
-        sh './build.sh'
+        script {          
+          ARTIFACT_REMOTE_DIRECTORY = "${env.BUILD_NUMBER}.0.0-${env.BRANCH_NAME.replace('/','-')}"
```

Comment:
rename ARTIFACT_REMOTE_DIRECTORY should be version number no?

Reply:
Correct. That's a more reasonable name. I'll change